presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotErrorCode.java

```diff
@@ -34,6 +34,7 @@
     PINOT_DECODE_ERROR(8, EXTERNAL),
     PINOT_INVALID_PQL_GENERATED(9, INTERNAL_ERROR),
     PINOT_INVALID_CONFIGURATION(10, INTERNAL_ERROR),
+    PINOT_DATA_FETCH_EXCEPTION(11, EXTERNAL, true),
```

Comment:
Is it possible to use the config `isMarkDataFetchExceptionsAsRetriable` here as the third param (retriable or not) ?

Reply:
Those are enums, I don't think we can set it with the pinotConfig.